inputgraph/inputgraph.go

```diff
@@ -287,9 +289,14 @@ func (l *loader) hashVersion(v spec.Version) {
 	l.hasher.HashJSONMarshalled(v)
 }
 
-func (l *loader) hashCommand(cmd spec.Command) {
-	cmd.SourceLocation = nil
-	l.hasher.HashJSONMarshalled(cmd)
+func (l *loader) hashCommand(c spec.Command) {
+	c.SourceLocation = nil
```

Comment:
I see that in one of the `hash*` functions, you simply pass through the Args, e.g.

```
func (l *loader) hashFor(f spec.ForStatement) {
	l.hasher.HashString("FOR")
	l.hasher.HashJSONMarshalled(f.Args)
}
```

but in others you set `SourceLocation = nil` and hash the entire command.

Any thoughts on making this consistent? I personally like the idea of explicitly hashing the Name and Args (like you did in `hashFor` -- since it makes it more clear what we're hashing on.

Reply:
It's likely better to be specific and reference individual fields. This prevents new fields from sneaking into the mix and so on.